docs/jinja.md

```diff
@@ -0,0 +1,71 @@
+# Generator framework based on Jinja template engine
+
+You can roll your own code generation approach with textX but sometimes it is
+good to have a predefined framework which is easy to get started with and only
+if you need something very specific later you can create your own code
+generator.
+
+Here, we describe a little framework based on
+[Jinja](https://jinja.palletsprojects.com/) template engine. This framework is
+implemented in [textX-jinja](https://github.com/textX/textX-jinja) project.
+
+The idea is simple. If you want to generate a set of files from your textX
+model(s) you create a folder which resembles the outline of the file tree you
+want to generate. Each file in your template folder can be a Jinja template
+(with `.jinja` extension, e.g. `index.html.jinja`), in which case target file
+will be created by running the template through the Jinja engine. If the file
+doesn't end with `.jinja` extension it is copied as is to the target folder.
+
+To call Jinja generator you use `textx_jinja_generator` function:
+
+```python
```

Comment:
This block of code is rendered into one line on my system...

Reply:
Is it rendered in one line in your editor or in the preview by `mkdocs serve`?

Reply:
In the preview (mkdocs serve). I will recheck in doubt...

Reply:
That's strange. Might be newlines issue?

Reply:
I recheck tonight or tomorrow. Surely I did a mistake...

Reply:
I have this effect for all pages presented with `mkdocs serve`... So it  must be an issue on my system. Just go ahead! Good work!